README.md

```diff
@@ -338,51 +338,86 @@ Once complete your base AWS resources should be in place
 
-3. Set the correct values for your new cluster config:
-```
-cd infra/terraform/global
-export KOPS_STATE_STORE=s3://`terraform output kops_bucket_name`
-export ENV_DOMAIN=`terraform output -module=cluster_dns dns_zone_domain`
-# Get the Client ID from Auth0 "kubectl-oidc" application e.g.
-export KUBECTL_OIDC_CLIENT_ID=P742wMtS4iiA6axtbPd2ygpOa64gZqGD
+  ```
+  OIDC:
+    ClientID: ""
+
+  instanceGroups:
+    masters:
+      machineType: ""
+
+    nodes:
+      machineType: ""
+      rootVolumeSize: ""
+      maxSize: ""
+      minSize: ""
+
+    bastions:
+      machineType: ""
+      maxSize: ""
+      minSize: ""
+  ```
 
-cd ../../../infra/terraform/platform
-../../kops/configure.sh $KOPS_STATE_STORE $ENVNAME $ENV_DOMAIN $KUBECTL_OIDC_CLIENT_ID
-```
+  | Key                    | Value                                                    |
+  |------------------------|----------------------------------------------------------|
+  | `OIDC.ClientID`        | OIDC client ID from the `kubectl-oidc` Auth0 application |
+  | `masters.machineType`  | EC2 instance type for master nodes, e.g. `m4.xlarge` |
+  | `nodes.machineType`    | EC2 instance type for worker nodes, e.g. `m4.xlarge` |
+  | `nodes.rootVolumeSize` | nodes' root disk size, in GB, e.g. `100` |
+  | `nodes.maxSize`        | max number of nodes to deploy |
+  | `nodes.minSize`        | min number of nodes to deploy. Specify the same value for `minSize` and `maxSize` to maintain a fixed size |
+  | `bastions.machineType` | EC2 instance type for SSH bastions, e.g. `t2.micro` |
+  | `bastions.maxSize`     | max number of bastions to deploy |
+  | `bastions.minSize`     | min number of bastions to deploy. Specify the same value for `minSize` and `maxSize` to maintain a fixed size |
+
+3. Generate a Kops values file containing additional info from Terraform:
+  ```
+  $ cd infra/kops
+  $ ./tf_output_to_kops_values.py
+  ```
+  This will output a `kops-tf-values.$ENVNAME.json` file in the current directory
 
-4. Ensure you've set the Kops state store environment variable (see previous step):
+4. Create a file containing Kops `ClusterSpec` and `InstanceGroup` resources from the template and values files:
   ```
-  $ echo $KOPS_STATE_STORE
-  s3://kops.analytics.justice.gov.uk
+  $ kops toolbox template \
+      --template cluster.tmpl.yml \
+      --values kops-tf-values.$ENVNAME.json \
+      --values config/$ENVNAME.yaml \
+      --output cluster.$ENVNAME.rendered.yml
   ```
 
-5. Plan Kops cluster resource creation:
+5. Set the Kops state store S3 bucket environment variable (see previous step):
+  ```
+  $ cd ../terraform/global
+  $ exports KOPS_STATE_STORE=s3://$(terraform output kops_bucket_name)
+  ```
+
+6. Plan Kops cluster resource creation:
 
   ```
-  cd ../../../infra/kops/clusters/$ENVNAME
-  kops create -f cluster.yml
-  kops create -f bastions.yml
-  kops create -f masters.yml
-  kops create -f nodes.yml
+  cd ../../kops/clusters/$ENVNAME
```

Comment:
I think this line is redundant now. That dir doesn't exist until you do the next line anyway.